pkg/controller/ttlafterfinished/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// TTLAfterFinishedSubsystem - subsystem name used for this controller.
+const TTLAfterFinishedSubsystem = "ttl_after_finished_controller"
+
+var (
+	// timeToDeletionSeconds tracks the time it took to delete the job since it
+	// became eligible for deletion.
+	TimeToDeletionSeconds = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      TTLAfterFinishedSubsystem,
+			Name:           "time_to_deletion_seconds",
+			Help:           "The time it took to delete the job since it became eligible for deletion",
+			StabilityLevel: metrics.ALPHA,
+			// Start with 100ms with the last bucket being [~2h, Inf).
+			Buckets: metrics.ExponentialBuckets(0.1, 2, 16),
```

Comment:
What's the value of having below seconds buckets, how about `metrics.ExponentialBuckets(2, 2, 15)` which would be inline with our other controllers. /cc @dashpole since he'll be looking at that specifically from instrumentation pov.

Reply:
I guess this depends on the cluster: a cluster that targets workloads with high churn of jobs may want a delete SLO below one second. This isn't harmful since it is just few extra buckets, and I think consistency with other controllers isn't really that important here.

Reply:
100ms seems quite reasonable to me, but if you feel strongly about number of buckets, maybe we can start at 200ms or 500ms

Reply:
but really, anything above 10min is probably way off any reasonable SLO

Reply:
I'm fine with lower boundary, but let's cap it sooner, if you're saying 10min, let's cap it at 14 which is slightly above 10 mins?

Reply:
done.